src/jobTree/jobStores/awsJobStore.py

```diff
@@ -420,9 +448,12 @@ def _getFileVersion( self, jobStoreFileID ):
         for attempt in retry_sdb( ):
             with attempt:
                 item = self.versions.get_attributes( item_name=jobStoreFileID,
-                                                     attribute_name='version',
+                                                     attribute_name=['version','bucketName'],
                                                      consistent_read=True )
-        return item.get( 'version', None )
+        bucketName = item.get('bucketName',None)
```

Comment:
I'm using a slight variation of this. 'bucketName' can actually be set while 'versions' is None if the file is registered with no content, but if 'bucketName' is None then 'versions' will also be None.